pages/_app.js

```diff
@@ -1,32 +1,34 @@
-import Script from 'next/script'
-import '../styles/globals.css'
-import * as gtag from '../lib/gtag'
+import '../styles/globals.css';
+import * as gtag from '../lib/gtag';
+import Head from 'next/head';
+import { useEffect } from "react";
+import { useRouter } from 'next/router';
+import loadCookieControl from 'components/CookieControl';
 
 function MyApp({ Component, pageProps }) {
+  const router = useRouter();
+  const apiKey = process.env.NEXT_PUBLIC_API_KEY;
+
+  useEffect(() => {
+    
+    loadCookieControl(apiKey);
+  }, [apiKey]);
+ 
   return (
     <>
-      <Script
-        strategy="afterInteractive"
-        src={`https://www.googletagmanager.com/gtag/js?id=${gtag.GA_TRACKING_ID}`}
-      />
-      <Script
-        id="gtag-init"
-        strategy="afterInteractive"
-        dangerouslySetInnerHTML={{
-          __html: `
-            window.dataLayer = window.dataLayer || [];
-            function gtag(){dataLayer.push(arguments);}
-            gtag('js', new Date());
-            gtag('config', '${gtag.GA_TRACKING_ID}', {
-              page_path: window.location.pathname,
-              cookie_flags: 'SameSite=None;Secure'
-            });
-          `,
-        }}
-      />
+      <Head>
+        <noscript>
+          <iframe
```

Comment:
I'm a bit confused by this, do we need this here as well as the stuff that gets added with `loadCookieControl`?